OneCommunityGlobal/HighestGoodNetworkApp
# Sav - Fix Profile Weekly Committed Hours Input

Fixes bug #12 Priority Medium  from USER PROFILE COMPONENT

# Description

On the profile page under the volunteering times tab, a user with edit permissions can change the weekly committed hours to unrealistic amounts. I've added a handle change on the input that should prevent users from adjusting those hours to anything below zero or anything above 168 (The maximum hours in one week). 

…

## Main changes
- Added handleChange function to check the status of values being passed to the input
- Give input min and max properties
- Alert user if an unrealistic value is attempted
…

## How to test:
1. Check into the current branch
2. Do `npm install` and `...` to run this PR locally
3. Log in as a user who can edit times on a profile (a user with canEdit: true)
4. Go to dashboard→ Profile→ Volunteering times→Weekly Committed Hours
5. Verify component WeeklyCommittedHours works properly and that users are not able to change their committed hours to an unrealistic number.

## Screenshots:
[PR MaxHours.webm](https://user-images.githubusercontent.com/98275089/224061105-1d618664-878e-4add-b821-38039c505df1.webm)
